mindmeld/constants.py

```diff
@@ -18,6 +18,21 @@
 BINARIES_URL = "https://binaries.mindmeld.com"
 DUCKLING_VERSION = "20200701"
 
+# ACTIVE LEARNING CONSTANTS
+ENTROPY_LOG_BASE = 2
+ACTIVE_LEARNING_RANDOM_SEED = 2020
```

Comment:
We can make this into an env var for people who want to set their own seed

Reply:
done